package.json

```diff
@@ -15,31 +15,49 @@
   },
   "devDependencies": {
     "@semantic-release/exec": "6.0.3",
+    "@tsconfig/node18": "2.0.1",
     "@types/node": "18.16.13",
+    "@types/semver": "7.5.0",
     "@types/shelljs": "0.8.12",
     "bats": "1.9.0",
     "bats-assert": "2.0.0",
     "bats-support": "0.3.0",
     "commander": "10.0.1",
```

Comment:
Should we open an issue to migrate from `commander` to `clipanion` for existing scripts? (I think it's just [`tools/utils.js`][1])

[1]: https://github.com/containerbase/base/blob/main/tools/utils.js

Reply:
sure

Reply:
will do later after this PR